content/cli/orch_cli/deployments.md

```diff
@@ -850,3 +874,174 @@ Node instances before rollback:
+  
+* `-l, --visibility TEXT`   Defines who can see the resource, can be set to one
+                            of ['private', 'tenant', 'global'] [default: tenant]
+  
+* `-t, --tenant-name TEXT`  The name of the tenant of the filter. If not
+                            specified, the current tenant will be used
+
+
+#### filters delete
+
+##### Usage
+
+`cfy deployments filters delete [OPTIONS] FILTER_ID` 
+
+Delete a deployments' filter.
+
+`FILTER-ID` is the filter's ID
+
+
+##### Optional flags
+
+* `-t, --tenant-name TEXT`  The name of the tenant of the filter. If not
+                            specified, the current tenant will be used
+
+
+#### filters get
+
+##### Usage
+
+`cfy deployments filters get [OPTIONS] FILTER_ID` 
+
+Get details for a single deployments' filter.
+
+`FILTER-ID` is the filter's ID
+
+
+##### Optional flags
+
+* `-t, --tenant-name TEXT`  The name of the tenant of the filter. If not
+                            specified, the current tenant will be used
+  
+#### Example
+
+{{< highlight bash  >}}
+$ cfy deployments filters get new_filter
+Getting info for deployments' filter `new_filter`...
+Requested deployments' filter info:
+	id:                        new_filter
+	visibility:                tenant
+	created_at:                2021-03-25 11:48:56.525 
+	updated_at:                2021-03-25 11:48:56.525 
+	tenant_name:               default_tenant
+	created_by:                admin
+	resource_availability:     tenant
+	private_resource:          False
+	labels_filter_rules:       "os=windows"
+	attrs_filter_rules:        "created_by starts-with bob"
+{{< /highlight >}}
+
+
+#### filters list
+
+##### Usage
+
+`cfy deployments filters list [OPTIONS]` 
+
+List all deployments' filters.
+
+##### Optional flags
+
+* `--sort-by TEXT`  Key for sorting the list
+  
+* `--descending`    Sort list in descending order [default: False]
+
+* `-t, --tenant-name TEXT`  The name of the tenant to list filters from.
+                            If not specified, the current tenant will be
+                            used. You cannot use this argument with
+                            arguments: [all_tenants]
+  
+* `-a, --all-tenants`   Include resources from all tenants
+                        associated with the user. You cannot use 
+                        this argument with arguments: [tenant_name].
+
+* `--search TEXT`   Search resources by name/id. The returned list will include 
+                    only resources that contain the given search pattern
+
+* `-o, --pagination-offset INTEGER`     The number of resources to skip; 
+                                        --pagination-offset=1 skips the first resource [default: 0]
+
+* `-s, --pagination-size INTEGER`   The max number of results to retrieve per page [default: 1000]
+
+
+#### filters update
+
+##### Usage
+
+`cfy deployments filters update [OPTIONS] FILTER_ID` 
+
+Update an existing deployments' filter's filter rules or visibility.
```

Comment:
Same comment as above - worth mentioning how the update takes place (what is updated)

Reply:
Done.